core/dbt/task/debug.py

```diff
@@ -102,13 +102,14 @@ def project_profile(self):
             return None
         return self.project.profile_name
 
-    def path_info(self):
-        open_cmd = dbt.clients.system.open_dir_cmd()
-        fire_event(OpenCommand(open_cmd=open_cmd, profiles_dir=self.profiles_dir))
-
     def run(self) -> bool:
+        # WARN: this is a legacy workflow that is not compatible with other runtime flags
```

Comment:
Flattened the code and commented it for the next person who wonders why this here in the `run` command.